RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -337,14 +341,22 @@
 mixedTripletStep = ClassifierMerger.clone()
 mixedTripletStep.inputClassifiers=['mixedTripletStepClassifier1','mixedTripletStepClassifier2']
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(mixedTripletStep, mixedTripletStepClassifier1.clone(
+	mva = dict(GBRForestLabel = 'MVASelectorMixedTripletStep_Phase1'),
+	src = 'mixedTripletStepTracks',
```

Comment:
the src= line is redundant and was not present in the pre-DNN version

Reply:
Redundant line removed